docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -89,8 +89,10 @@ means it has not yet been bound to a PersistentVolumeClaim.
 
 The next step is to create a PersistentVolumeClaim. Pods use PersistentVolumeClaims
 to request physical storage. In this exercise, you create a PersistentVolumeClaim
-that requests a volume of at least three gibibytes that can provide read-write
-access for at least one Node.
+that requests a volume of at least three gigabytes that can provide read-write
+access for at least one Node. In order to bind the PersistentVolumeClaim to the
+PersistentVolume we created in the previous step, we also must [disable dynamic
```

Comment:
IMO, the point here is not to disable dynamic provisioning. Kubernetes now always checks for existing PVs before provisioning a new one (unless you use alpha annotation and you don't and even if you would, alpha is not supported in 1.7 any longer). The reason for explicit `pvs.spec.storageClassName: ""` is to get around a default StorageClass. You create your PV without any class and you want your PVC to explicitly require a PV that has no StorageClass. Otherwise default StorageClass (say `foo`) is applied to the PVC, Kubernetes won't find available PV with class `foo` and it will provision a new one.